subcommand/inject-connect/command.go

```diff
@@ -62,7 +62,7 @@ type Command struct {
 	http    *flags.HTTPFlags
 
 	consulClient *api.Client
-	clientset    *kubernetes.Clientset
+	clientset    kubernetes.Interface
```

Comment:
Changing this for tests. Also we should be using interfaces rather than structs.